### PR TITLE
Transaction info

### LINKED ISSUE #53: Transaction info by its hash
Enter hash - see info.

### PR DESCRIPTION
**Pull request by yaroshchuk8**
Closes #53

### CODE REVIEW

File: src/crypto.js [hunk 1]
```diff
@@ -109,6 +109,64 @@ class Crypto {
...
+      4: 'ltc',
+    };
+
+    console.log('\nList of cryptos:');
+    for (const key in cryptoNames) {
+      console.log(`${key}. ${cryptoNames[key]}`);
+    }
+
+    const chosenCrypto = await promised.question('\nEnter the number' +
+      ' of crypto from the list above you\'d to like to input hash of: \n');
```
Comment (ILabiak) [inline src/crypto.js:132]: you can use ` ` instead of using ' ' twice.

**Review by ILabiak — COMMENTED** [review]
Use ` ` instead of using ' ' twice.

File: src/crypto.js [hunk 2]
```diff
@@ -109,6 +109,64 @@ class Crypto {
     }
   }
 
+  async transactionInfo() {
```
Comment (akaeyuhi) [inline src/crypto.js:112]: You need to move this method to exchange.js accroding to new file structure

File: src/crypto.js [hunk 3]
```diff
@@ -109,6 +109,64 @@ class Crypto {
...
+    };
+    const abbreviation = {
+      1: 'btc',
+      2: 'dash',
+      3: 'doge',
+      4: 'ltc',
+    };
+
+    console.log('\nList of cryptos:');
+    for (const key in cryptoNames) {
```
Comment (akaeyuhi) [inline src/crypto.js:127]: Don`t use for in cycles if you don`t mind working with properties in __proto__

File: src/crypto.js (hunk 1 above)
Comment (akaeyuhi) [reply src/crypto.js:132]: Yeah, instead of concatenation use formatted string

File: src/crypto.js [hunk 4]
```diff
@@ -109,6 +109,64 @@ class Crypto {
...
+    ];
+    if (Object.prototype.hasOwnProperty.call(info, 'error')) {
+      console.log(`${red}Wrong hash${green}`);
+      return;
+    } else if (Object.prototype.hasOwnProperty.call(info, 'confirmed')) {
+      keys.push('confirmed');
+      outputKeys.push('Confirmed at');
+    } else console.log('\nTransaction isn\'t confirmed yet :C');
+
+    for (let i = 0; i < keys.length; i++) {
```
Comment (akaeyuhi) [inline src/crypto.js:164]: Use for of

File: src/crypto.js [hunk 5]
```diff
@@ -109,6 +109,64 @@ class Crypto {
...
+    const outputKeys = [
+      '\nSatoshis sent',
+      'Fee in satoshis',
+      'Transaction size in bytes',
+      'Transaction preference',
+      'Received at',
+    ];
+    if (Object.prototype.hasOwnProperty.call(info, 'error')) {
+      console.log(`${red}Wrong hash${green}`);
+      return;
```
Comment (akaeyuhi) [inline src/crypto.js:158]: Throw Error or return it

**Review by akaeyuhi — CHANGES_REQUESTED** [review]

File: src/crypto.js [hunk 6]
```diff
@@ -109,6 +109,64 @@ class Crypto {
...
+    for (const key in cryptoNames) {
+      console.log(`${key}. ${cryptoNames[key]}`);
+    }
+
+    const chosenCrypto = await promised.question('\nEnter the number' +
+      ' of crypto from the list above you\'d to like to input hash of: \n');
+    const hash = await promised.question('\nEnter the hash of ' +
+      'transaction you\'d like to get info about: \n');
+
+    if (hash.length !== 64) {
```
Comment (tshemsedinov) [inline src/crypto.js:136]: Magic number (compare with named constant)

File: src/crypto.js [hunk 7]
```diff
@@ -109,6 +109,64 @@ class Crypto {
...
+      'Transaction preference',
+      'Received at',
+    ];
+    if (Object.prototype.hasOwnProperty.call(info, 'error')) {
+      console.log(`${red}Wrong hash${green}`);
+      return;
+    } else if (Object.prototype.hasOwnProperty.call(info, 'confirmed')) {
+      keys.push('confirmed');
+      outputKeys.push('Confirmed at');
+    } else console.log('\nTransaction isn\'t confirmed yet :C');
```
Comment (tshemsedinov) [inline src/crypto.js:162]:
```suggestion
    } else {
      console.log('\nTransaction isn\'t confirmed yet :C');
    }
```
Just for code style consistency

**Pull request merged**